build_tools/packaging/linux/build_package.py

```diff
@@ -58,6 +64,56 @@ class PackageConfig:
 
 ################### Debian package creation #######################
 def create_deb_package(pkg_name, config: PackageConfig):
+    """Function to create deb package
+    Invoke the creation of versioned and non versioned package
+    Move the DEB packages to destination directory
+
+    Parameters:
+    pkg_name : Name of the package to be created
+    config: Configuration object containing package metadata
+
+    Returns: None
+    """
+    print(inspect.currentframe().f_code.co_name)
+    print(f"Package Name: {pkg_name}")
+
+    create_nonversioned_deb_package(pkg_name, config)
+    create_versioned_deb_package(pkg_name, config)
+    move_packages_to_destination(pkg_name, config)
+
+
+def create_nonversioned_deb_package(pkg_name, config: PackageConfig):
+    """Function to create non versioned deb package
+    Get package details and generate control file
+    Non versioned Packages are meta packages.
+    Create deb package
+
+    Parameters:
```

Comment:
grammar nit: This docstring summary a bit all over the place. Can you rephrase it?

https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
Ack

Reply:
Addressed the same